src/app/views.py

```diff
@@ -432,6 +433,36 @@ def licenseNamespaceInformation(request, licenseId):
         )
 
 
+def ntia_check(request):
+    """ View for ntia checker tool
+    returns ntia_conformance_checker.html template
+    """
+    if request.user.is_authenticated or settings.ANONYMOUS_LOGIN_ENABLED:
+        context_dict={}
+        if request.method == 'POST':
+            core.initialise_jpype()
+            result = core.ntia_check_helper(request)
+            jpype.detachThreadFromJVM()
+            context_dict = result.get('context', None)
+            status = result.get('status', None)
+            response = result.get('response', None)
+            message = result.get('message', None)
+
+            if response and status:
+                return HttpResponse(response, status=status)
```

Comment:
Why are returning just http response when repsonse and status are successfull? Users would see a blank screen with http status displayed. Also, will this work with async javascript calls from UI?

Reply:
@linynjosh  Can you share the screenshot/video of the new page?

Reply:
<img width="1401" alt="Screen Shot 2022-10-14 at 7 00 55 PM" src="https://user-images.githubusercontent.com/63569843/195963958-8130ba7e-434a-48bf-a2f2-65260e6c4150.png">
<img width="1401" alt="Screen Shot 2022-10-14 at 7 01 42 PM" src="https://user-images.githubusercontent.com/63569843/195963986-03494176-6e73-4453-a0cf-a4b3257c1c14.png">